telcell/crunchers/dummycruncher.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+from itertools import pairwise
+from typing import Any, Iterable, Iterator, Mapping, Tuple
+
+from telcell.data.models import Track
+from telcell.data.utils import extract_intervals, split_track_by_interval
+
+
+def dummy_cruncher(tracks: Iterable[Track]) \
+        -> Iterator[Tuple[Track, Track, Mapping[str, Any]]]:
+    """
+    Dummy data crunchers that takes each consecutive pair of tracks `(track_a,
+    track_b)` and splits `track_a` into slices for each day. Yields a 3-tuple
```

Comment:
er wordt nu niet meer gesplitst per dag maar per uur

Reply:
Hiermee is de start om 5:00 's nachts ook niet meer zo logisch, wat maken we daar van?